### ISSUE TITLE
Posts in user repo are not showing in profile or feeds

### ISSUE THREAD

**Issue by mushion22** [bluesky-social/social-app#7241]
### Steps to Reproduce

This is for a specific user - @sydneystations.bsky.social (they are a friend of mine, posting on their behalf) 

Since yesterday, no posts that they've posted are showing in apps using https://public.api.bsky.app/xrpc/app.bsky.feed.getAuthorFeed (ie it doesn't show up in the profile for themselves or any other followers using bsky.app web or mobile). 

The posts do show up when quering the repo on the PDS, eg https://puffball.us-east.host.bsky.network/xrpc/com.atproto.repo.listRecords?collection=app.bsky.feed.post&repo=did%3Aplc%3A6ijpli7zizf2oqf3rf3qcuhy

All posts from at://did:plc:6ijpli7zizf2oqf3rf3qcuhy/app.bsky.feed.post/3lds4lnknr22a onward are not showing in their author feed via the bsky app view (ie public.api.bsky.app) and thus not displaying in any of the official/3rd-party apps that use that to show feeds. 

My own account happens to be on the same PDS and is working fine so it does not appear to be a PDS-wide issue. 

### What platform(s) does this occur on?

iOS, Android, Web (Desktop), Web (Mobile)

### What version of the app are you using?

1.96.1

**Comment by Apreslabiere** [comment]
Exactly the same issue for my account : https://bsky.app/profile/apreslabiere.bsky.social 

My likes / repost or comments does not show or appear since several days. My posts does not seem to appear to anyone either even if I can see them under the third tab of my profile ("Média" in french). It seems that someone else visiting my profile cannot see them. I m using the web client, I did try on different browsers and I logged out and in several times. I am running a mac Sequoia 15.2.

Picture of the two latest "posts" that no one could see and that I can see in the third tab of my profile. 
<img width="611" alt="image" src="https://github.com/user-attachments/assets/87b6fcd1-7bf1-4122-b374-519f8895b84a" />

**Comment by mushion22** [comment]
I suggested my friend deactivates his account.

A couple of hours later it's still visible in apps using the Bsky appview, as is his feed up to the same point as prior to deactivation (ie missing the posts from past couple of days).

Seems like his profile and feed are being cached in the relay or appview and not refreshed?

**Comment by mushion22** [comment]
Ok I was tinkering some more and it gets weirder. 

With his account deactivated: 

1. Calling his PDS com.atproto.repo.listRecords with app.bsky.feed.post and his DID, unauthenticated: Could not find repo. 
![image](https://github.com/user-attachments/assets/7a4418fb-9fb6-4357-bd2d-bb6a670ab413)

2. Calling his PDS app.bsky.feed.getAuthorFeed with auth (from my account, not his): Shows same as public app. 
![image](https://github.com/user-attachments/assets/49407b9e-3725-4c56-93c7-9e15df00b78e)

I'm still learning about the bsky architecture, but looks like the PDS is proxying to the PDS Entryway or the App view here? Not sure. 

3. Calling public.api.bsky.app pp.bsky.feed.getAuthorFeed unauthenticated: returns same result as (2). 

![image](https://github.com/user-attachments/assets/c34609a8-8b88-4de6-8a14-424643cad122)

Though I note there are some headers indicating a CDN hit, however adding Cache-Control: no-cache and thus forcing a cdn miss still has the same response body. 

![image](https://github.com/user-attachments/assets/5d8376a9-7759-46e2-b067-feebd92e3079)

![image](https://github.com/user-attachments/assets/e2ee0df6-669c-40be-9b3c-ad88705f4b06)

4. The PDS describeRepo correctly describes it as deactivated. 
![image](https://github.com/user-attachments/assets/2878d2ac-9424-4755-b935-af21fe7c08ab)

**Comment by mushion22** [comment]
I had a look at @Apreslabiere profile and there is something similar there. 

1. GET https://public.api.bsky.app/xrpc/app.bsky.feed.getAuthorFeed?actor=did:plc:5ywigech4j7n4gosnapfi4cc -> same feed as visible on apps (missing posts). 
![image](https://github.com/user-attachments/assets/7e771220-6ee8-4bfd-bff2-92f8a2b74778)


2. GET https://chanterelle.us-west.host.bsky.network/xrpc/com.atproto.repo.listRecords?collection=app.bsky.feed.post&repo=did:plc:5ywigech4j7n4gosnapfi4cc -> has latest posts

![image](https://github.com/user-attachments/assets/af9b8d39-59c9-4c98-8a10-8c764f1eb433)

An added twist: doing a resolveHandle for him via bsky.social gets a 400. 

![image](https://github.com/user-attachments/assets/0e04ac62-07a9-49f7-8e60-515f101e1f7a)

But works fine via my PDS (puffball - I had first just looked it up in the webapp and saw this via devtools)

![image](https://github.com/user-attachments/assets/cc401904-4144-43fc-b579-d1931dc1201a)

**Comment by gaearon** [comment]
Raised to the backend folks.

**Comment by mushion22** [comment]
Same issue happening in #7288 

Latest post in getAuthorFeed from public.api.bsky.social is at://did:plc:z5w5syndfjwr6imvcqyd3fjk/app.bsky.feed.post/3ldqv33khdc2q while latest in the repo on their PDS (lepista.us-west) is at://did:plc:z5w5syndfjwr6imvcqyd3fjk/app.bsky.feed.post/3lea3sllb3s2g 
From public API: 
![image](https://github.com/user-attachments/assets/4ae9e56d-697d-484b-9933-4b3d61e480a8)
From PDS API:
![image](https://github.com/user-attachments/assets/01e0b863-5631-48b7-a43a-c55395c6abb1)

**Comment by mushion22** [comment]
This is affecting a bunch of accounts

https://www.reddit.com/r/BlueskySocial/comments/1hnd9pb/is_the_app_not_working/

https://www.reddit.com/r/BlueskySocial/comments/1hn0zxj/question_ive_made_3_posts_that_register_as_live/

https://www.reddit.com/r/BlueskySocial/comments/1hmmazz/my_posts_arent_published_since_20_december/

https://www.reddit.com/r/BlueskySocial/comments/1hmbs3n/whats_with_accounts_locking_up/

**Comment by Apreslabiere** [comment]
Modification of profile info does not seem to work either...

**Comment by JazRav** [comment]
Changing handles if your account is locked up like this allows for 2 handles to point to the same account DID when @'d by someone else.

**Comment by davetroy** [comment]
I can confirm that this same issue is affecting my account, @davetroy.com. I have been unable to post, reply, like, follow, unfollow, etc since December 20. I have opened three separate tickets with support with no response, canned or otherwise. Supportive followers have posted on Bsky and X tagging support and other Bsky company accounts, without any response so far.

Am not upset; just want to see this resolved, and apparently it’s affecting many other users. My account was created on May 1, 2023 and lives on the ‘Porcini’ personal data server.
![IMG_0506](https://github.com/user-attachments/assets/914efbbc-ce8d-4198-a8d9-ac8843466741)

**Comment by oosterhagen** [comment]
It's New Year. Hopefully the people at BlueSky will find a resolution next year.

**Comment by kirahowe** [comment]
I seem to be experiencing the same thing as @kirahowe.com, not sure exactly when it started.

**Comment by mushion22** [comment]
> This should be fixed, see [#7288 (comment)](https://github.com/bluesky-social/social-app/issues/7288#issuecomment-2574150733) onwards.

Yep the account in my original post is now displaying the missing posts and is working correctly. Thanks!
